src/frontend/src/utils/styleUtils.ts

```diff
@@ -347,6 +348,9 @@ export const nodeIconToDisplayIconMap: Record<string, string> = {
   ChatVertexAI: "VertexAI",
   ChevronsUpDownIcon: "ChevronsUpDown",
   ClearMessageHistory: "FileClock",
+  CleanlabEvaluator: "Cleanlab",
+  CleanlabRAGEvaluator: "Cleanlab",
+  CleanlabRemediator: "Cleanlab",
```

Comment:
The icon name is correctly set to `Cleanlab` in all your python files so this unnecessary